src/tracker-component.js

```diff
@@ -330,7 +335,11 @@ export const Tracker = (config? : Config = defaultTrackerConfig) => {
 
             return trackCartEvent(config, 'setCart', newCart);
         },
-        removeFromCart: (data : RemoveCartData) => trackCartEvent(config, 'removeFromCart', data),
+        removeFromCart: (data : RemoveCartData) => {
+            const newCart = composeCart('remove', data);
```

Comment:
Do we need test coverage for this portion? Making sure that removeFromCart modifies the localStorage as expected

Reply:
I did some manual testing, it works as-expected for subsequent calls.

You're right though. I'll write some tests for the 'parent' composeCart method and throw a bunch of different get/set/remove calls at it to make sure everything is squared away.